crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -195,9 +195,36 @@ impl BaseRoomInfo {
                 let mut o_ev = o_ev.clone();
                 o_ev.content.set_created_ts_if_none(o_ev.origin_server_ts);
```

Comment:
Can we please not use abbreviations, presumably `o_ev` is `original_event`. `Ok(u)` can easily be `Ok(user_id)`. I see no benefit in shortening them this way, especially as it's not consistent with the rest of the scope.

Reply:
`o_ev` was there before this PR, so I'll leave the task of changing it to a later PR to keep this one focused.  But I will rename the `u` variables that were added in this PR.

Reply:
Fair 👍

Reply:
[`8932070`](https://github.com/matrix-org/matrix-rust-sdk/pull/3836/commits/89320701a401558d327693da16e9ae09c5093af8)